app/uk/gov/hmrc/cipphonenumberverification/services/VerifyHelper.scala

```diff
@@ -85,18 +87,24 @@ abstract class VerifyHelper @Inject()(otpService: OtpService, auditService: Audi
   private def sendPasscode(phoneNumberAndOtp: PhoneNumberAndOtp)
                           (implicit hc: HeaderCarrier) = govUkConnector.sendPasscode(phoneNumberAndOtp) map {
     case Left(error) => error.statusCode match {
-      case SERVICE_UNAVAILABLE =>
-        logger.error(error.getMessage())
+      case INTERNAL_SERVER_ERROR =>
+        logger.error(error.getMessage)
         BadGateway(Json.toJson(ErrorResponse(EXTERNAL_SERVICE_FAIL, "Server currently unavailable")))
+      case BAD_REQUEST | FORBIDDEN =>
+        logger.error(error.getMessage)
+        ServiceUnavailable(Json.toJson(ErrorResponse(EXTERNAL_API_FAIL, "External server currently unavailable")))
+      case TOO_MANY_REQUESTS =>
+        logger.error(error.getMessage)
+        TooManyRequests(Json.toJson(ErrorResponse(MESSAGE_THROTTLED_OUT, "The request for the API is throttled as you have exceeded your quota")))
       case _ =>
-        logger.error(s"Gov Notify failure - to be covered by CAV-163")
-        InternalServerError(Json.toJson(ErrorResponse(EXTERNAL_SERVICE_FAIL, "to be covered by CAV-163")))
+        logger.error(error.getMessage)
+        Result.apply(ResponseHeader(error.statusCode), HttpEntity.NoEntity)
```

Comment:
same as comment above, is this correct?

Reply:
As I mentioned we will still send the error.status code